mne/evoked.py

```diff
@@ -854,6 +854,17 @@ def grand_average(all_evoked, interpolate_bads=True):
     return grand_average
 
 
+def _check_evokeds_ch_names_times(all_evoked):
+    evoked = all_evoked[0]
+    ch_names = evoked.ch_names
+    for ev in all_evoked[1:]:
+        assert ev.ch_names == ch_names, ValueError(
+            "%s and %s do not contain " "the same channels" % (evoked, ev))
```

Comment:
`assert` statements are skipped with `python -O`, if this is meant to actually prevent some user error (rather than internal consistency check) this needs to be done the standard way